### ISSUE TITLE
LPC/FWH support?

### ISSUE THREAD

**Issue by Split7fire** [dword1511/stm32-vserprog#4]
On old project page (https://github.com/dword1511/serprog-stm32vcp) it is said that "Support LPC / FWH flash via parallel programming. [WIP]" but that about stm32-vserprog? Does it support LPC/FWH at hte moment?

**Comment by dword1511** [comment]
No. In the old version it was marked as work in progress (WIP) but I never had time to finish it.

**Comment by Split7fire** [comment]
Thanks for fast response,
so, LPC/FWH support in that project is working anyway? (read/write)

**Comment by dword1511** [comment]
Unfortunately it is not.

**Comment by Split7fire** [comment]
ok, thanks for clarification, I think this issue can be closed now.

**Comment by swiftgeek** [comment]
Relevant fork/derivative work:
https://github.com/wosk/stm32-vserprog-lpc
